resolwe/flow/views/data.py

```diff
@@ -153,7 +152,7 @@ def duplicate(self, request, *args, **kwargs):
                 "Data objects with the following ids not found: {}".format(', '.join(map(str, missing_ids)))
             )
 
-        # TODO support ``inherit_collections``
+        # TODO support ``inherit_collection``
```

Comment:
We should support `inherit_collection` and `inherit_entity`. We are waisting queries doing this manually afterwards.